assets/js/modules/adsense/components/setup/SetupAccountCreate.js

```diff
@@ -100,24 +104,30 @@ export default function SetupAccountCreate() {
 						parseAccountID( existingTag )
 					) }
 				{ ! existingTag &&
-					sprintf(
-						/* translators: %s: user email address */
-						__(
-							'Already use AdSense? Add %s as a user to an existing AdSense account.',
-							'google-site-kit'
+					createInterpolateElement(
+						sprintf(
+							/* translators: %s: user email address */
+							__(
+								'Already use AdSense? Add %s as a user to an existing AdSense account. <SupportLink />',
```

Comment:
Maybe better something like this:
```suggestion
								'Already use AdSense? Add %s as a user to an existing AdSense account. <a>Learn more</a>',
```